### PR TITLE
US003 view capability role

### PR DESCRIPTION
**Pull request by caitlinbulloch**
**View the Capability a Role belongs to**

- Updated the view 'list-jobs.html' to show capability name alongside job role.
- Updated the model 'job' file to include capability name. 

Screenshot 2023-12-14 at 09 40 34

### CODE REVIEW

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Code looks good! I think you have a conflict you need to resolve! Also no one else on your team has reviewed this before the tech mentors 🙃 

I would also rename your PR title to include the same ticket number as your planner ie. US003 instead of 003. Some integration tools rely on them matching. 

Also it's a shame this PR doesn't include any UI tests. Give us a shout if these are causing issues and we can hopefully help 😄

**Review by Cormac-F — APPROVED** [review]
LGTM!

**Comment by jkelso662** [comment]
Checks are failing! Worth getting that sorted before we do a code review 😄

**Comment by CharlesM988** [comment]
@jkelso662 @lullabelle UI testing has been added and the errors have been resolved. 

Screenshot 2023-12-19 at 11 58 46
Screenshot 2023-12-19 at 12 33 39

File: Test/JobServiceTest.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+            .get('/job-roles')
+      
+    });
+});
+
+
+
+
+
+
```
Comment (jkelso662) [inline Test/JobServiceTest.js]: Watch out for all this blank space!

File: Test/ui/CapabilityTest.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      await driver.findElement(By.linkText("Employee")).click();
+
+      await driver.wait(until.elementLocated(By.id("jobs-container")), 5000);
+
+      await driver.executeScript("arguments[0].scrollIntoView(true);", await driver.findElement(By.id("jobs-container")));
+
+      await driver.wait(until.elementLocated(By.css("#job-table tbody tr:first-child td:last-child")), 5000);
+
+      const capabilityName = await driver.findElement(By.css("#job-table tbody tr:first-child td:last-child")).getText();
+
```
Comment (jkelso662) [inline Test/ui/CapabilityTest.js:22]: I guess we have picked up this double spacing from Shauns notes? haha

**Review by jkelso662 — APPROVED** [review]
This looks really good! Can't wait to see these selenium tests actually running 👯  Added a couple of comments re spacing to please bare in mind next PR.

File: Test/ui/CapabilityTest.js (hunk 2 above)
Comment (CharlesM988) [reply Test/ui/CapabilityTest.js:22]: > I guess we have picked up this double spacing from Shauns notes? haha

@jkelso662 Maybeeeeeee

**Pull request merged**